src/main/java/io/github/reconsolidated/zpibackend/features/item/dtos/ItemDto.java

```diff
@@ -30,8 +30,8 @@ public class ItemDto {
     private Integer amount;
     private Integer availableAmount;
     private Double mark;
-    private Integer earliestStartHour = 0;
-    private Integer latestEndHour = 24;
+    private Integer earliestStartHour = 10;
+    private Integer latestEndHour = 16;
```

Comment:
earliestStartHour = 23 latestEndHour = 0. What about minutes? reverse < in earliest condition

Reply:
done